builds/jazz-build-module/service-configuration-data-loader.groovy

```diff
@@ -305,6 +305,13 @@ def loadServiceConfigurationData() {
       updateConfigValue("{user_pool_id}", config_loader.JAZZ.PLATFORM.AWS.COGNITO.USER_POOL_ID)
       updateConfigValue("{user_client_id}", config_loader.JAZZ.PLATFORM.AWS.COGNITO.CLIENT_ID)
       updateConfigValue("{region}", region)
+      updateCoreAPI()
+      updateConfigValue("{conf-region}", region)
+      updateConfigValue("{jazz_admin}", config_loader.JAZZ.STACK_ADMIN)
+      updateConfigValue("{jazz_admin_creds}", config_loader.JAZZ.STACK_PASSWORD)
+      sh "sed -i -- 's#\"{vault_is_enabled}\"#${config_loader.TVAULT.IS_ENABLED}#g' ./config/dev-config.json"
```

Comment:
Please check if it replaces `vault_is_enabled` with false when this feature is disabled. In other words, ensure that `config_loader.TVAULT.IS_ENABLED = false` when Vault integration is not enabled.

Reply:
Sure. Its by default false in installer .
https://github.com/tmobile/jazz-installer/blob/secret-mgmt/installer/terraform/provisioners/cookbooks/jenkins/files/default/jazz-installer-vars.json#L195